expertsystem/data.py

```diff
@@ -32,12 +33,29 @@ def __eq__(self, other: object) -> bool:
             return self.__value == other.value
         return self.__value == other
 
+    def __gt__(self, other: Any) -> bool:
+        if isinstance(other, Parity):
+            return self.value > other.value
+        return self.value > int(other)
+
+    def __ge__(self, other: Any) -> bool:
+        return self > other or self == other
+
+    def __lt__(self, other: Any) -> bool:
+        return not self >= other
+
+    def __le__(self, other: Any) -> bool:
+        return not self > other
+
     def __int__(self) -> int:
         return self.value
 
     def __neg__(self) -> "Parity":
         return Parity(-self.value)
 
+    def __hash__(self) -> int:
+        return self.__value
+
```

Comment:
Side note: I had problems with the (previously) missing `__hash__` method when creating a `set` of `tuples` with particle info (in this case a set of tuples of parities with spins).

Reply:
I can't really follow the need for this functionality?

Reply:
I tried to do something like this:
```python
>>> from expertsystem import io
p.parity, p.spin) for p in pdg.values()}
>>> pdg = io.load_pdg()
>>> {(p.parity, p.spin) for p in pdg.values()}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 1, in <setcomp>
TypeError: unhashable type: 'Parity'
```

Reply:
Ok, I'm not sure what the use case for doing something like this would be. I guess a user wants to reduce the particle to some important qns. Btw, you could still do `(p.parity.value,...`

Reply:
That is true. Or, alternatively, `int(p.parity)`.

Reply:
See also https://github.com/ComPWA/expertsystem/pull/271#discussion_r494829408